project_tests.sh

```diff
@@ -5,7 +5,7 @@ set -e
 : "${dependencies:?Need to set dependencies environment variable}"
 if [ "$dependencies" = "lowest" ]; then
     composer update --prefer-lowest --no-interaction
-    proofreader src/
+    proofreader --no-phpcpd src/
```

Comment:
eheh is this trying to tell you something?

Reply:
From slack:

```
hi chris, let me know when you have time to discuss the api-sdk-php
11:08
The issue is with duplicated code between Collection and RegionalCollection classes
11:08
the normalizer classes are affected also

chris  11:11 AM
...tempted to say just turn off PHPCPD… (edited) 
11:11
Code reviews are more effective, and doesn’t have the strict problem.

nathan:house_with_garden:  11:11 AM
ok, we might hit the same issue with Journal
```

Reply:
Happy to consider a different approach and avoid the duplication. Would prefer to pair on it to improve chances of acceptance.

Reply:
I think the `ForSubject` trait has addressed most of this duplication. The rest is made up of properties on Collections and PromotionalCollections that are quite similar, but to reduce that we would probably end up extracting traits that don't have a clear name or a domain concept. I'd favor keeping clarity by duplicating those properties, which is the current state of this PR. :+1: